### ISSUE TITLE
BATT voltage measurement pin on ESP32-C3 pin 5 has error

### ISSUE THREAD

**Issue by sayanee** [hutscape/capsicum#5]
## Description

`Pin 5` on ESP32-C3 is ADC2. And when using it to measure analog voltage, it gives the following error:

```
E (14210) ADC: ADC2 is no longer supported, please use ADC1. Search for errata on espressif website for more details. You can enable ADC_ONESHOT_FORCE_USE_ADC2_ON_C3 to force use ADC2
E (14215) ADC: adc2_get_raw(750): adc unit not supported
```

## Steps to Reproduce

Measure analog value from the pin

```
#define BATTERY_MEASURE_PIN 5
...
int rawValue = analogRead(BATTERY_MEASURE_PIN);
```

## Expected Behavior

There shouldn't be any error.

## Actual Behavior

That line of code gives an error:

```
E (14210) ADC: ADC2 is no longer supported, please use ADC1. Search for errata on espressif website for more details. You can enable ADC_ONESHOT_FORCE_USE_ADC2_ON_C3 to force use ADC2
E (14215) ADC: adc2_get_raw(750): adc unit not supported
```

## Environment

- Arduino firmware
- [Espressif errata](https://www.espressif.com/sites/default/files/documentation/esp32-c3_errata_en.pdf)

## Screenshot

<img width="772" alt="Screenshot 2024-04-11 at 8 38 29 PM" src="https://github.com/hutscape/capsicum/assets/413879/deab17fd-c666-4146-b753-e643978721ef">

## References

`GPIO05` is `ADC2` from [ESP32-C3-Mini-1 Datasheet, page 11](https://www.espressif.com/sites/default/files/documentation/esp32-c3-mini-1_datasheet_en.pdf).

<img width="746" alt="image" src="https://github.com/user-attachments/assets/0c41d4d9-03cb-4a65-b9ec-0f5b50465133">

**Comment by sayanee** [comment]
## Fix schematic

- Change `BATT` from `GPIO5` (ADC2) to `GPIO0` (ADC1)
- Change `R10` to `10k` ohms

![schematic](https://github.com/hutscape/capsicum/assets/413879/8f2e59ff-ac91-4fd0-8af8-8b10186899a3)

## Fix layout

- Cut trace from `R9` to test point `GPIO05`
- Add new bodge wire from `R9` to test point `GPIO0`

![layout](https://github.com/hutscape/capsicum/assets/413879/beb23a4d-af62-48c3-bab6-8c5a07471502)

**Comment by sayanee** [comment]
`R9` connected to `GPIO0` 

### Schematic

![image](https://github.com/hutscape/capsicum/assets/413879/a69236c0-6460-48c5-abc2-15a32dc4f0a1)


### Layout

![image](https://github.com/hutscape/capsicum/assets/413879/d34efa62-c3ed-41c2-a812-756b7712ac9c)

**Comment by sayanee** [comment]
Closed with 61efc72173ea0ea0a2b0cf0940e66e0a65f15c36
